content/influxdb/v2.3/reference/cli/influx/scripts/retrieve.md

```diff
@@ -0,0 +1,39 @@
+---
+title: influx scripts retrieve
+description: The `influx scripts retrieve` command retrieves a script in InfluxDB.
```

Comment:
I think this might be misleading, since the `retrieve` command does not return the flux contained in the script, only the name, description, and language.

Reply:
Gotcha, you're right. What about:

> The `influx scripts retrieve` command retrieves invokable script information from InfluxDB.

Reply:
👍 done